examples/test/test_i_multiple_kms_cmk_regions.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+"""Unit test suite for the encryption and decryption using multiple KMS CMKs in multiple regions example."""
+
+import botocore.session
+import pytest
+
+from ..src.multiple_kms_cmk_regions import multiple_kms_cmk_regions
+
+# from .examples_test_utils import get_cmk_arn
+from .examples_test_utils import static_plaintext
+
+
+pytestmark = [pytest.mark.examples]
+
+
+def test_multiple_kms_cmk_regions():
+    plaintext = static_plaintext
+    cmk_arn1 = "arn:aws:kms:us-west-2:658956600833:alias/EncryptDecrypt"
```

Comment:
Rather than hard-coding these, add another helper in `integration_test_utils` and trickle that up through.
Also, when you do that, update the integration test readme.

Please use this environment variable: `AWS_ENCRYPTION_SDK_PYTHON_INTEGRATION_TEST_AWS_KMS_KEY_ID_2`.

I'll update our CI configurations to have this set to the correct value.
Related, the value that is stored here *must* be a CMK Arn, not an alias Arn (I'll make sure it is in CI).
We might relax that requirement once we switch over to keyrings, but for now various things require it.